sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj.nuget.dgspec.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
this file shouldn't be needed. So are other misc. json files below.

Reply:
Agreed, I don't know why these are here.

Reply:
I didn't add them manually. Swagger generation through autorest created those. What should I do? Rever the unnecessary ones?